### PR TITLE
Hello, world!

### PR DESCRIPTION
**Pull request by Kaspiman**
Do you plan to further develop this project?

I would like to use it in product development it. Code quality is very high already, but maybe i can improve it. Thanks.

### CODE REVIEW

**Comment by root-aza** [comment]
Hi. Yep!

File: src/ExceptionInterceptor/DoctrinePingConnectionExceptionInterceptor.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ public function __construct(
 
     public function isRetryable(Throwable $e): bool
     {
-        if ($e::class == EntityManagerClosed::class) {
+        if ($e instanceof EntityManagerClosed) {
```
Comment (root-aza) [inline src/ExceptionInterceptor/DoctrinePingConnectionExceptionInterceptor.php:32]: EntityManagerClosed final class. why instanceof?

https://github.com/doctrine/orm/blob/2.16.x/lib/Doctrine/ORM/Exception/EntityManagerClosed.php

Comment (root-aza) [reply src/ExceptionInterceptor/DoctrinePingConnectionExceptionInterceptor.php:32]: @Kaspiman cc

Comment (Kaspiman) [reply src/ExceptionInterceptor/DoctrinePingConnectionExceptionInterceptor.php:32]: cc

**Pull request merged**